part-spec/pinPaths.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$id": "https://github.com/edatasheets/edatasheets.github.io/blob/main/part-spec/pinPaths.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "pinPaths": {
+    "type": "object",
+    "required": [
+      "partPinPaths"
+    ],
+    "properties": {
+      "numberofpinPaths": {
```

Comment:
numberOfPinPaths

Mix of camelCase here

Reply:
fixed